src/watchful/client.py

```diff
@@ -441,6 +441,18 @@ def records(csv_: str) -> Optional[Dict]:
     return api("records", data=csv_, content_type="text/csv")
 
 
+def col_flags(flag: str, columns: List[bool]) -> Optional[Dict]:
```

Comment:
should this change to `col_flag` also since IIRC that's where the verb landed?

Reply:
Nice! Done